### PR TITLE
[flake8] Disable Y065 for now

### PR DESCRIPTION
**Pull request by srittau**
This unblocks PyCQA/flake8-pyi#475

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -13,7 +13,8 @@ per-file-ignores =
   # Y023: Alias typing as typing_extensions
   # Y026: Have implicit type aliases
   # Y053: have literals >50 characters long
-  *_pb2.pyi: Y021, Y023, Y026, Y053
+  # Y065: TODO: Remove bare Incompletes
+  *_pb2.pyi: Y021, Y023, Y026, Y053, Y065
```
Comment (AlexWaygood) [inline .flake8:17]: This only ignores Y065 for files ending in `_pb2.pyi`

**Comment by AlexWaygood** [comment]
FWIW, I think we can just merge https://github.com/PyCQA/flake8-pyi/pull/475 without this change (CI isn't failing on the PR, and we know _why_ there's a big diff). We can easily add the Y065 ignore  at the same time as bumping our flake8-pyi pin at typeshed

**Review by srittau — COMMENTED** [review]
Fix config

File: .flake8 (hunk 1 above)
Comment (srittau) [reply .flake8:17]: That happens when editing too many branches at the same time ...

**Comment by srittau** [comment]
Sure, let's do that.

**Pull request closed without merging**